.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDatePeriodParser.cs

```diff
@@ -1073,7 +1073,7 @@ private DateTimeResolutionResult ParseOneWordPeriod(string text, DateObject refe
                     var swift = 0;
                     if (nextMatch.Success)
                     {
-                        if (nextMatch.Groups[Constants.AfterGroupName].Success)
+                        if (nextMatch.Groups[Constants.AfterGroupName].Success || nextMatch.Value.Equals("下下", StringComparison.Ordinal))
```

Comment:
Why is this needed? It would be better to define a constant in the YAML for the value with a good name, instead of hardcoding a substring here.

Reply:
A code comment here explaining why this is needed would also be very useful.

Reply:
> Why is this needed? It would be better to define a constant in the YAML for the value with a good name, instead of hardcoding a substring here.

Thanks for the comments, finish updating the code.